packages/editor-ui/src/App.vue

```diff
@@ -17,6 +18,11 @@ import { useSettingsStore } from '@/stores/settings.store';
 import { useHistoryHelper } from '@/composables/useHistoryHelper';
 import { useStyles } from './composables/useStyles';
 
+// Polyfill crypto.randomUUID
+if (!('randomUUID' in crypto)) {
+	Object.defineProperty(crypto, 'randomUUID', { value: uuid });
+}
```

Comment:
Just to be sure. The `window.crypto` object is always defined, and it's just `randomUUID` that's missing?